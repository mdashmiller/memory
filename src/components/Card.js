import styled from 'styled-components'

const Card = styled.div`
	height: 80px;
	width: 72px;
	border-radius: 5px;
	background-color: red;
`

export default Card
