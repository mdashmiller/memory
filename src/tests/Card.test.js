import React from 'react'
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'
import Card from '../components/Card'

describe('Card', () => {

	it('renders without crashing', () => {
		const div = document.createElement('div')
		ReactDOM.render(<Card>Test</Card>, div)
		ReactDOM.unmountComponentAtNode(div)
	})

	test('has a valid snapshot', () => {
		const component = renderer.create(
			<Card>Test</Card>
		)
		let tree = component.toJSON()
		expect(tree).toMatchSnapshot()
	})

})
