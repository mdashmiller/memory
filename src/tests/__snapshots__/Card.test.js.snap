// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card has a valid snapshot 1`] = `
<div
  className="sc-bdVaJa kPlaWI"
>
  Test
</div>
`;
